components/function-controller/README.md

```diff
@@ -1,184 +1,39 @@
+| **BUILD_TIMEOUT**                               | 30m                                                               | Build timeout for Tekton TaskRuns. [Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".](https://golang.org/pkg/time/#ParseDuration)                                                                                           |
+| **CONTROLLER_CONFIGMAP**                        | fn-config                                                         | Name of the ConfigMap containing information about available runtimes, function types and default values                                                                                                                                   |
+| **CONTROLLER_CONFIGMAP_NS**                     | default                                                           | Namespace of that ConfigMap                                                                                                                                                                                                                |
+| **CONTROLLER_DOCKER_REGISTRY_FQDN**             | function-controller-docker-registry.kyma-system.svc.cluster.local | Fully qualified domain name of Docker Registry service                                                                                                                                                                                     |
+| **CONTROLLER_DOCKER_REGISTRY_PORT**             | 5000                                                              | Docker Registry service port through which communication is conducted                                                                                                                                                                      |
+| **CONTROLLER_DOCKER_REGISTRY_EXTERNAL_ADDRESS** | https://registry.kyma.local                                       | Domain address through which connection to Docker Registry is facilitated                                                                                                                                                                  |
+| **CONTROLLER_IMAGE_PULL_SECRET_NAME**           | regcred                                                           | Name of the secret image pull secret which contains hashed credentials to Docker Registry                                                                                                                                                  |
+| **CONTROLLER_TEKTON_REQUESTS_CPU**              | 350m                                                              | Specifies minimal amount of CPU assigned to TaskRun building lambda image. Follow [this](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu) document to get to know available values    |
```

Comment:
```suggestion
| **CONTROLLER_TEKTON_REQUESTS_CPU**              | `350m`                                                              |  Minimum amount of CPU assigned to the TaskRun to build the lambda image. See [this](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu) document for available values.   |
```

Reply:
I removed `Specifies` to be consistent with other descriptions in the table. I also changes `Follow` into `See `as I guess you rather follow some steps in instruction ...